vnext/Microsoft.ReactNative/PropertySheet.props

```diff
@@ -14,4 +14,14 @@
       <Value>$(CppWinRTVerbosity)</Value>
     </BuildMacro>
   </ItemGroup>
+    
+  <ItemDefinitionGroup Condition="'$(Configuration)'=='WinUI3'">
```

Comment:
>Configuration [](start = 37, length = 13)

Could you please introduce a different flag instead of using Configuration? The Configuration ideally must be only Debug and Release. There are too many other dependencies on it. 
All other optional components depend on a specific property name.
If you want to have something configurable from the IDE, then it is better to use the C++ CPS rules and have it set in the object properties. E.g. see the C++/WInRT settings.

Reply:
Please ignore this comment. I looked at the other changes and cannot think about better way to achieve the same. It sounds that having a custom configuration in this case is a right approach.